src/util/inconsistencies.py

```diff
@@ -209,6 +210,7 @@ def verify_uoms(
     include_archived_products=FROM_ENV,
     auto_fix=FROM_ENV,
     ids=None,
+    raise_if_inconsistent=False,
```

Comment:
I would instead take a policy as argument instead of a boolean option.
```suggestion
    on_missmatch="report",
```

The other accepted value would be `"fail"`.